compose.yaml

```diff
@@ -0,0 +1,15 @@
+name: app
+
+services:
+  app:
+    build: .
+    image: ml
+    container_name: app
+    ports:
+      - "80:8000"
+    command: 
+      [
+        "fastapi", 
+        "run", 
+        "main.py"
+      ]
```

Comment:
Move this command back to the `CMD` in the Dockerfile.
It ensures our Docker image can run independently without needing compose file (in k8s environment for example)

Reply:
Discussed